### PR TITLE
Adding event 9154 - detect order opened in editor

### LINKED ISSUE #157: Event ID 2100 reported when user opens an order
When I open an order on WooCommerce, this event is reported, which is seeing / reporting the order as a post:

image

At the moment we do not have any event IDs for when a user opens an order, but it is something we should create now as well.

### PR DESCRIPTION
**Pull request by DannyWPWS**
Fixes #157

### CODE REVIEW

File: wp-security-audit-log/custom-sensors/WSAL_Sensors_WooCommerce.php [hunk 1]
```diff
@@ -244,6 +244,82 @@ public function HookEvents() {
...
+			}
+		}
+
+		if ( ! empty( $referrer ) && strpos( $referrer, $current_path ) !== false ) {
+			// Ignore this if we were on the same page so we avoid double audit entries.
+			return $post;
+		}
+
+		if ( ! empty( $post->post_title ) && ! self::was_triggered_recently( 9154 ) ) {
+			$event = 9154;
```
Comment (DannyWPWS) [inline wp-security-audit-log/custom-sensors/WSAL_Sensors_WooCommerce.php:310]: :warning: **Warning**: Equals sign not aligned with surrounding assignments; expected 7 spaces but found 1 space (*Generic.Formatting.MultipleStatementAlignment.NotSameWarning*).

**Review by DannyWPWS — COMMENTED** [review]
**phpcs** scanning turned up:

:warning: 1 warning 

***

**hashes-api**-scanning skipped

**Comment by DannyWPWS** [comment]
GitHub API communication error. Please contact a human. (commit-ID: 5fe48e082726f52841b298ad0157128dd355eff6).

**Comment by DannyWPWS** [comment]
GitHub API communication error. Please contact a human. (commit-ID: 0eb2dc753aa72dd336a2771d52b50b93ed1b6041).

**Review by sdobreff — APPROVED** [review]

**Pull request merged**